webapp/sources/rudder/rudder-web/src/main/elm/sources/Accounts/JsonDecoder.elm

```diff
@@ -54,9 +55,19 @@ decodeAcl =
         |> required "verb" string
 
 
+parseToken : String -> ( Token )
+parseToken str =
+  case str of
+    "2" -> Token.Hashed
+    "1" -> Token.ClearText
+    _   -> Token.New str
+
```

Comment:
okay, I would have used another "tokenVersion" attribute to avoid confusion on this information sensitive field, but since it is an internal endpoint I think it is okay